install.ts

```diff
@@ -0,0 +1,203 @@
+import {
+  ErrorKind,
+  Process,
+  ProcessStatus,
+  args,
+  chmod,
+  copyFile,
+  env,
+  exit,
+  lstat,
+  platform,
+  symlink,
+  writeFile,
+  removeAll,
+  rename,
+  run,
+  makeTempDir
+} from "deno";
+
+import { mkdirp } from "https://deno.land/x/std/mkdirp/mkdirp.ts";
+import { join, win32 } from "https://deno.land/x/path/index.ts";
+
+const WIN32: boolean = platform.os === "win";
+
+async function prep_wunzip(
+  temp_dir: string,
+  zip_file: string,
+  dest_dir: string
+): Promise<string[]> {
+  const script_file: string = join(temp_dir, "wunzip.vbs");
+  const script_data: Uint8Array = new TextEncoder().encode(
+    `
+    set objShell = CreateObject("Shell.Application")
+    set FilesInZip=objShell.NameSpace("${zip_file}").items
+    objShell.NameSpace("${dest_dir}").CopyHere(FilesInZip)
+    Set objShell = Nothing
+  `
```

Comment:
Instead of doing this with Visual Basic Script, how about using PowerShell for the unzipping? With PowerShell, it would be `Expand-Archive C:\file.zip -DestinationPath C:\directory`, and you wouldn't need to write a temporary script file, but instead pass the command to PowerShell via `powershell.exe -Command {CommandTextHere}`.

Reply:
@MarkTiedemann thanks, that is way less clutter! switched to ps unzipping.